src/service/viewer-impl.js

```diff
@@ -263,6 +264,12 @@ export class Viewer {
         || this.isWebviewEmbedded_
         || !ampdoc.isSingleDoc());
 
+    /**
+     * Whether the AMP document is embedded in a Chrome Custom Tab.
+     * @private @const {boolean}
+     */
+    this.isCustomTab_ = this.win.location.search.indexOf('amp_agsa=1') != -1;
```

Comment:
Nit: Let's use "CCT" to reference. Also, there are a few existing ivars e.g. `isWebviewEmbedded_`. 

So let's use `this.isCctEmbedded_`.